codegen/service.go

```diff
@@ -247,6 +248,16 @@ func (ms *ModuleSpec) SetDownstream(
 			"Service %q does not have method %q\n", serviceName, methodName,
 		)
 	}
+
+	if e.IsDummyEndpoint {
+		funcSpec := method.CompiledThriftSpec
+		err := method.setDummyTypeConverters(funcSpec, reqTransforms, headersPropagate, respTransforms, dummyReqTransforms, h)
+		if err != nil {
+			return err
+		}
+		return nil
```

Comment:
Some message on the error? or just dispense with the error generation

Reply:
having an error message will help with debugging a lot.